packages/nimbus/src/components/range-calendar/range-calendar.mdx

```diff
@@ -1,93 +1,156 @@
-
-  return (
-    <Stack direction="horizontal" alignItems="center">
-      {sizes.map(size => (
-        <RangeCalendar key={size} size={size}>'{size}' RangeCalendar</RangeCalendar>
-      ))}
-    </Stack>
-  )
-}
-```
+[Figma library](https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?m=auto&node-id=5818-15651)
 
-### Variants
+## Variables
 
-Available variants.
+Get familiar with the features.
 
-```jsx-live
-const App = () => {
-
-  const variants = ['solid', 'subtle', 'outline', 'ghost', 'plain'];
-
-  return (
-    <Stack direction="horizontal">
-      {variants.map(variant => (
-        <RangeCalendar key={variant} variant={variant}>'{variant}' RangeCalendar</RangeCalendar>
-      ))}
-    </Stack>
-  )
-}
-```
+### Visual options
 
-### Colors
+#### Date range
 
-[Explain usage with different colors/semantics.]
+Allow users to select a range of dates.
 
 ```jsx-live
-const App = () => {
-  const variants = ["solid", "subtle", "outline", "ghost", "link", "plain"];
-  const colors = ["neutral", "primary", "info", "positive", "warning", "critical"];
-
-  return (
-    <Stack>
-      {colors.map((color) => (
-        <Stack key={color} direction="horizontal">
-          {variants.map((variant) => (
-            <RangeCalendar colorPalette={color} key={variant} variant={variant}>
-              '{variant}' RangeCalendar
-            </RangeCalendar>
-          ))}
-        </Stack>
-      ))}
-    </Stack>
-  );
-};
-``` */}
+const App = () => <RangeCalendar aria-label="Enter dates" />
+```
+
+## Guidelines
+
+Calendar guidelines focus on creating intuitive and accessible interfaces for
+viewing, navigating, and interacting with dates and events.
+
+### Best practices
+
+- **Clear visual hierarchy:** Distinguish between days, weeks, months, and
+  years. Highlight the current date and any selected dates.
+- **Intuitive navigation:** Provide clear controls to move between different
+  time periods (days, weeks, months, years).
+- **Easy date range selection:** For applications requiring date ranges, provide
+  an intuitive way to select start and end dates, often with visual
+  highlighting.
+- **Clear labels and instructions:** Provide clear labels for navigation
+  controls and any interactive elements within the calendar.
+
+### Usage
+
+A calendar is used to visually display and manage time-based information,
+allowing users to view, schedule, and interact with dates. A common way this
+component is used is within the popover, and paired with the date input.
+
+> [!TIP]\
+> When to use
+
+- **Scheduling:** When users need to select specific dates and time ranges.
+- **Visualizing time-based data:** To display data or to filter data in a
+  corresponding visual.
+- **Date range selection:** When users need to select a start and end date for a
+  period.
+- **Browsing dates:** When users need to navigate through different days, weeks,
+  or months to find a specific date or time.
```

Comment:
```suggestion
- **Browsing dates:** When users need to navigate through different days, weeks,
  or months to find a specific date.
```
 or...specific dates?
lies if were just referencing the Range Calendar/ DateRangeCalendar/Calendar Range itself.